File: median-of-two-sorted-arrays/median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int>res;
        for(auto num:nums1){
            res.push_back(num);
        }
        for(auto num:nums2){
            res.push_back(num);
        }
        sort(res.begin() , res.end());
        int mid = res.size() /2;
        if(res.size()%2==1){
            return res[mid];
        }
        else{
            return( res[mid-1]+res[mid])/2.0;
        }
        
    }
};

File: median-of-two-sorted-arrays/median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        nums1 = nums1+nums2
        nums1 = sorted(nums1)
        mid = len(nums1)//2
        if len(nums1)%2==1:
            return nums1[mid]
        else:
            return (nums1[mid-1]+nums1[mid])/2.0